alembic/versions/2023_03_10_0331-143a068da0ed_initial_migration.py

```diff
@@ -196,20 +235,97 @@ def upgrade():
         sa.Column("table", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
         sa.Column("cost", sa.Float(), nullable=False),
         sa.Column("id", sa.Integer(), nullable=False),
-        sa.Column("catalog_id", sa.Integer(), nullable=True),
         sa.Column("database_id", sa.Integer(), nullable=False),
-        sa.ForeignKeyConstraint(
-            ["catalog_id"],
-            ["catalog.id"],
-            name=op.f("fk_table_catalog_id_catalog"),
-        ),
         sa.ForeignKeyConstraint(
             ["database_id"],
             ["database.id"],
             name=op.f("fk_table_database_id_database"),
         ),
         sa.PrimaryKeyConstraint("id", name=op.f("pk_table")),
     )
+    op.create_table(
+        "tagnoderelationship",
+        sa.Column("tag_id", sa.Integer(), nullable=False),
+        sa.Column("node_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["node_id"],
+            ["node.id"],
+            name=op.f("fk_tagnoderelationship_node_id_node"),
+        ),
+        sa.ForeignKeyConstraint(
+            ["tag_id"],
+            ["tag.id"],
+            name=op.f("fk_tagnoderelationship_tag_id_tag"),
+        ),
+        sa.PrimaryKeyConstraint(
+            "tag_id",
+            "node_id",
+            name=op.f("pk_tagnoderelationship"),
+        ),
+    )
+    op.create_table(
+        "columnattribute",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("attribute_type_id", sa.Integer(), nullable=True),
+        sa.Column("column_id", sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["attribute_type_id"],
+            ["attributetype.id"],
+            name=op.f("fk_columnattribute_attribute_type_id_attributetype"),
+        ),
+        sa.ForeignKeyConstraint(
+            ["column_id"],
+            ["column.id"],
+            name=op.f("fk_columnattribute_column_id_column"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_columnattribute")),
+        sa.UniqueConstraint(
+            "attribute_type_id",
+            "column_id",
+            name=op.f("uq_columnattribute_attribute_type_id"),
+        ),
+    )
+    op.create_table(
+        "cuberelationship",
```

Comment:
This is actually autogenerated by alembic from the sqlmodel `CubeRelationship` class. Since cube definitions are just node definitions, they share the `node` table and this is just a two column lookup table used as a "link model". Would a better name be something like "CubeLookup" or "CubeLink"?

Reply:
@samredai you can override the default table names generated for models using the `__tablename__` attribute (something like that).

Reply:
Nice! Alright I'll keep the classname the same and change the table name to "cube"

Reply:
That worked nicely, thanks @betodealmeida!